tools/challenge-helper-scripts/insert-challenge.ts

```diff
@@ -0,0 +1,47 @@
+import ObjectID from 'bson-objectid';
+import { prompt } from 'inquirer';
+import { challengeTypeToTemplate } from './helpers/get-challenge-template';
+import { newChallengePrompts } from './helpers/new-challenge-prompts';
+import { getProjectPath } from './helpers/get-project-info';
+import { getMetaData, updateMetaData } from './helpers/project-metadata';
+import { createChallengeFile } from './utils';
+import { getChallengeOrderFromMeta } from './helpers/get-challenge-order';
+
+const insertChallenge = async () => {
+  const path = getProjectPath();
+
+  const options = await newChallengePrompts();
+
+  const challenges = getChallengeOrderFromMeta();
+
+  const challengeAfter = (await prompt({
+    name: 'id',
+    message: 'Which challenge should come AFTER this new one?',
+    type: 'list',
+    choices: challenges.map(([id, title]) => ({
+      name: title,
+      value: id
+    }))
+  })) as { id: string };
```

Comment:
```suggestion
  const challengeAfter = await prompt<{id: string}>({
    name: 'id',
    message: 'Which challenge should come AFTER this new one?',
    type: 'list',
    choices: challenges.map(([id, title]) => ({
      name: title,
      value: id
    }))
  })
```
There's not much in it, but this seems a little cleaner.